hs_core/views/__init__.py

```diff
@@ -1079,36 +1079,39 @@ def create_resource_select_resource_type(request, *args, **kwargs):
 @login_required
 def create_resource(request, *args, **kwargs):
     # Note: This view function must be called by ajax
-
     ajax_response_data = {'status': 'error', 'message': ''}
     resource_type = request.POST['resource-type']
     res_title = request.POST['title']
     resource_files = request.FILES.values()
     source_names = []
-    irods_fnames = request.POST.get('irods_file_names')
-    federated = request.POST.get("irods_federated").lower() == 'true'
+
+    # NOTE: We want to remove any access that users have to upload a file from
+    # iRods, so this following code is irrelevant to a front-facing create_resource method
+
+    # irods_fnames = request.POST.get('irods_file_names')
+    # federated = request.POST.get("irods_federated").lower() == 'true'
     # TODO: need to make REST API consistent with internal API. This is just "move" now there.
     fed_copy_or_move = request.POST.get("copy-or-move")
 
-    if irods_fnames:
-        if federated:
-            source_names = irods_fnames.split(',')
-        else:
-            user = request.POST.get('irods-username')
-            password = request.POST.get("irods-password")
-            port = request.POST.get("irods-port")
-            host = request.POST.get("irods-host")
-            zone = request.POST.get("irods-zone")
-            try:
-                upload_from_irods(username=user, password=password, host=host, port=port,
-                                  zone=zone, irods_fnames=irods_fnames, res_files=resource_files)
-            except utils.ResourceFileSizeException as ex:
-                ajax_response_data['message'] = ex.message
-                return JsonResponse(ajax_response_data)
-
-            except SessionException as ex:
-                ajax_response_data['message'] = ex.stderr
-                return JsonResponse(ajax_response_data)
+    # if irods_fnames:
+    #     if federated:
+    #         source_names = irods_fnames.split(',')
+    #     else:
+    #         user = request.POST.get('irods-username')
+    #         password = request.POST.get("irods-password")
+    #         port = request.POST.get("irods-port")
+    #         host = request.POST.get("irods-host")
+    #         zone = request.POST.get("irods-zone")
+    #         try:
+    #             upload_from_irods(username=user, password=password, host=host, port=port,
+    #                               zone=zone, irods_fnames=irods_fnames, res_files=resource_files)
+    #         except utils.ResourceFileSizeException as ex:
+    #             ajax_response_data['message'] = ex.message
+    #             return JsonResponse(ajax_response_data)
+
+    #         except SessionException as ex:
+    #             ajax_response_data['message'] = ex.stderr
+    #             return JsonResponse(ajax_response_data)
```

Comment:
I'm trying to think of a strategy that would work best to minimize any effort around keeping this project in sync with hydroshare (aka, minimize merge conflicts). That essentially means: change as few files as possible. Can this same bug be fixed by passing the hidden variables for  `irods_file_names` and `irods_federated`? If so I think it'd be better to go that route: one less file changed in MyHPOM means one less file to cause conflicts, and the reason it isn't working is b/c we were a little overzealous ripping out HTML in the create-resource.html template anyway...

If that doesn't work for some reason, then...rather than commenting out this code, remove it. Git will track the fact that it was there...

Reply:
That is a good argument to change as few files as possible.  Passing the hidden variables seems to work as well.  Depending on what happens after merging in PR#99, I will make changes or open new PR to just re-create the commenting out of iRods stuff.